core/arch/arm/include/kernel/cache_helpers.h

```diff
@@ -31,6 +32,24 @@ void dcache_op_all(unsigned long op_type);
 void dcache_op_level1(unsigned long op_type);
 void dcache_op_level2(unsigned long op_type);
 void dcache_op_level3(unsigned long op_type);
+
+/*
+ * Get system cache line size from ARM system configuration registers.
+ */
+static inline uint32_t dcache_get_line_size(void)
+{
+	uint32_t value = 0;
+
+#ifdef ARM64
+	value = read_ctr_el0();
```

Comment:
We could add a `#define read_ctr() read_ctr_el0()` in `<arm64.h>` to avoid this ifdef. We've already done so for a few other registers.

Reply:
Ok